### PR TITLE
chore(appearance-ui): move to docs (VIV-1839)

### PR DESCRIPTION
**Pull request by RichardHelm**

### CODE REVIEW

File: .github/workflows/_lint.yml [hunk 1]
```diff
@@ -11,15 +11,20 @@ jobs:
...
+
       - uses: actions/cache@v3
         id: cache
         with:
-          path: node_modules/
+          path: |
+            node_modules/
+            apps/vue-docs/node_modules/
+            libs/storybook-vue2/node_modules/
+            libs/storybook-vue3/node_modules/
```
Comment (YonatanKra) [inline .github/workflows/_lint.yml:22]: Why do they have their own `node_modules`?

Comment (RichardHelm) [reply .github/workflows/_lint.yml:22]: They need different versions of some packages

**Review by rachelbt — COMMENTED** [review]
I changed one of the connotation colors, ran a ui test - got lots of changes in component but no new images were generated in the appearance-ui :(

**Comment by rachelbt** [comment]
> I changed one of the connotation colors, ran a ui test - got lots of changes in component but no new images were generated in the appearance-ui :(

after rechecking and running `npx nx run docs:e2e` all is OK

**Review by rachelbt — APPROVED** [review]

**Comment by RichardHelm** [comment]
@rachelbt Sure, I added it to the UI test readme now

**Review by rachelbt — APPROVED** [review]
Great!

**Pull request merged**